EE Getting started: add SSH forwarding (#39)
ansible/ecosystem-documentation: docs/getting_started_ee/run_execution_environment.md

    ansible-navigator run test_remote.yml -i inventory --execution-environment-image postgresql_ee:latest --mode stdout --pull-policy missing --enable-prompts -u student -k -K
    ```

> ## SSH keys forwarding

Reviewer: ```suggestion
## SSH key forwarding
```

> If you run Ansible promptlessly using SSH-keys-based authentication, use `ssh-agent` to forward the SSH keys to your EE:

Reviewer: ```suggestion
If you run Ansible with SSH-keys-based authentication to avoid authentication prompts, use `ssh-agent` to forward the SSH keys to your EE:
```

> If you run Ansible with SSH-keys-based authentication to avoid authentication prompts, use `ssh-agent` to forward the SSH keys to your EE:

Reviewer: ```suggestion
If you run Ansible with SSH-key-based authentication to avoid authentication prompts, use `ssh-agent` to forward the SSH keys to your EE:
```

``` bash
eval $(ssh-agent)
ssh-add </path/to/your/private/key>
```

> To test the above playbook, add the `remote_user: <your_ssh_user>` field to the play in the playbook and run:

Reviewer: ```suggestion
To test the above playbook, add the `remote_user: <your_ssh_user>` field to the play and then run:
```

> To test the above playbook, add the `remote_user: <your_ssh_user>` field to the play and then run:

Reviewer: ```suggestion
To test the preceding playbook, add the `remote_user: <your_ssh_user>` field to the play and then run:
```
